lib/config/httpRoutes.js

```diff
@@ -280,6 +280,7 @@ module.exports = [
   {verb: 'delete', url: '/roles/:_id', controller: 'security', action: 'deleteRole'},
   {verb: 'delete', url: '/users/:_id', controller: 'security', action: 'deleteUser'},
   {verb: 'delete', url: '/credentials/:strategy/:_id', controller: 'security', action: 'deleteCredentials'},
+  {verb: 'delete', url: '/_revokeTokens/:_id', controller: 'security', action: 'revokeTokens'},
```

Comment:
I would find 
`DELETE /users/:_id/tokens` more RESTful (not using gh suggestion here as it will break the functional tests & doc).